decidim-participatory_processes/config/locales/uk.yml

```diff
@@ -265,7 +265,6 @@ uk:
           processes:
             one: "%{count} процес"
             few: "%{count} процеси"
-            many: "%{count} процесів"
```

Comment:
You were right @josepjaume. Needs to be fixed in the crowdin side.

Reply:
Yeah, but I'm not sure how to tackle it since I cannot find any config related to this case:

https://support.crowdin.com/configuration-file/

Maybe we should add the `many` key to all languages and ignore it in all of them except Ukrainian?

Reply:
I hate being right. @mrcasals is digging into the issue, let's see if we can talk with the crowdin crowd (pun intended).